roles/srcds/files/addons/sourcemod/scripting/nativevotes_mapchooser.sp

```diff
@@ -1251,9 +1268,17 @@ void CreateNextVote()
 	// tempMaps is a resolved map list
 	ArrayList tempMaps = new ArrayList(ByteCountToCells(PLATFORM_MAX_PATH));
 	
-	for (int i = 0; i < g_MapList.Length; i++)
+	// use curated mappool or
+	ArrayList maps = new ArrayList();
```

Comment:
I don't think you need to initialise this here, as you'd be instantiating a new `ArrayList` that you'd never close and thus have a (small) memory leak.

```suggestion
	ArrayList maps;
```

Reply:
![image](https://user-images.githubusercontent.com/9057156/173419378-d8a8b99e-d817-448c-9775-8683ec4dc8f5.png)